test/test_user.py

```diff
@@ -0,0 +1,234 @@
+		self.assertEqual(response.status_code,405)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result'],'Invalid input (Empty data)')
+
+	def testEmail(self):
+		"""
+		test if invalid email
+		"""
+		email={
+		'firstName':'kwame',
+		'lastName': 'Asiago',
+		'email': '@.',
+		'dob': '2/3/2018',
+		'Location': 'kasarani',
+		'password': 'password'
+		}
+		response = self.test.post('/auth/signup',
+			headers=self.headers,data=json.dumps(email))
+		self.assertEqual(response.status_code,405)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result'],'Invalid data email')
+
+	def testDate(self):
+		"""
+		test if invalide date
+		"""
+		date={
+		'firstName':'kwame',
+		'lastName': 'Asiago',
+		'email': 'kwame@gmail.com',
+		'dob': '2/33/2018',
+		'Location': 'kasarani',
+		'password': 'password'
+		}
+		response = self.test.post('/auth/signup',
+			headers=self.headers,data=json.dumps(date))
+		self.assertEqual(response.status_code,405)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result'],'Invalid Date')
+
+	def testWhiteSpace(self):
+		"""
+		test if contains white space only
+		"""
+		whitespace={
+		'firstName':' ',
+		'lastName': 'Asiago',
+		'email': 'kwame@gmail.com',
+		'dob': '2/33/2018',
+		'Location': 'kasarani',
+		'password': 'password'
+		}
+		response = self.test.post('/auth/signup',
+			headers=self.headers,data=json.dumps(whitespace))
+		self.assertEqual(response.status_code,405)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result'],'Invalid input(Only whitespace)')
+
+	def testExist(self):
+		"""
+		test if user exist
+		"""
+		response = self.test.post('/auth/signup',
+			headers=self.headers,data=json.dumps(self.userData))
+		self.assertEqual(response.status_code,409)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result'],'Email already exist')
+
+	def testDataType(self):
+		response = self.test.post('/auth/signup',
+			headers=self.headers,data=json.dumps(self.userData))
+		self.assertEqual(response.status_code,405)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result'],'Invalid data type')
+
+
+class SignInTest(unittest.TestCase):
+	def setUp(self):
+		self.userData = {
+		'email':'kwameasiago',
+		'password':'password'
+		}
+		self.headers = {'Content-type':'application/json'}
+		self.test = app.test_client()
+
+	def tearDown(self):
+		self.userData = None
+		self.headers = None
+		self.test = None
+
+	def testEmptyEmail2(self):
+		"""test if email is empty
+		"""
+		response = self.test.post('/auth/login',
+			headers=self.headers,data=json.dumps(self.userData))
+		self.assertEqual(response.status_code,405)
+		data = json.loads(response.get_data().decode('utf-8'))
+		self.assertEqual(data['result':'Invalid data-empty email'])
+
+	def testEmptyPassword2(self):
```

Comment:
Check on method naming style, it should be more descriptive. Check out this resource as a refresher: https://dzone.com/articles/7-popular-unit-test-naming